syntax = "proto2";

package opentxs.proto;
option java_package = "org.opentransactions.proto";
option java_outer_classname = "OTVerificationGroup";
option optimize_for = LITE_RUNTIME;

import public "VerificationIdentity.proto";

message VerificationGroup {
    optional uint32 version = 1;
    repeated VerificationIdentity identity = 2;
}
